ci/pipeline.yml

```diff
@@ -867,14 +869,49 @@ jobs:
     - get: cg-provision-repo
       trigger: true
       passed: [plan-bootstrap-gov-root]
-  - task: plan
+  - task: apply
     file: pipeline-tasks/terraform-apply.yml
     input_mapping: {terraform-templates: cg-provision-repo}
     params:
       <<: *gov-root-params
       TERRAFORM_ACTION: apply
   - *notify-slack
 
+- name: plan-bootstrap-gov-prd-plat
```

Comment:
I'd love to have a general chat about how we're going to manage our AWS accounts in terraform, but I don't think it's necessary right now — what you have in this PR looks like a good start 👍

Reply:
Sure. I'm happy to have that conversation anytime